Campaign_Manager/tests.py

```diff
@@ -1,3 +1,81 @@
 from django.test import TestCase
+from .models import *
+from Character_Builder.models import *
 
-# Create your tests here.
+# Campaign test cases
+class CampaignCreateTestCase(TestCase):
+	def setUp(self):
+		Campaign.objects.create(campaignName="The Mountain")
+
+	def test_campaign_name(self):
+		campaign = Campaign.objects.get(campaignName="The Mountain")
```

Comment:
instead of self.assertEqual, put the Campaign.objects.get(campaignName="The Mountain") in a "try:, except:" block and use self.fail() if the getter returns an exception. Same goes for the next test

Reply:
all good, this has been added in